pom.xml

```diff
@@ -3,16 +3,18 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.videoplaza</groupId>
     <artifactId>kafka-connect-pubsub</artifactId>
-    <version>0.15.9</version>
+    <version>0.16.5</version>
 
     <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Downgrading to Java 8? Problems running this in GCP?

Reply:
No that's our builds mostly, which is temporary. On the other hand since this is an open source project it might be worth it to keep on lower versions.